Automattic/bugomattic
# Add the first full reporting flow

#### What Does This PR Add/Change?

This adds the first fully functioning reporting flow!

The main pieces that changed to make that happen were...

- Create implementations of all the "steps" of the reporting flow
- Add redux state to track the currently "active" step
- Updating the existing forms to start from the current issue values, and to advance to the next step on continue
- A loading indicator while the reporting config is loading
- Updating `app.tsx` to use all the real stuff!
- Deleting old debug components

#### Testing Instructions

Boot up the app with `yarn start` -- the main reporting flow should now be in a functional state!

Also can run `yarn test:once`

#### Issues



Related to #  
Closes #
